### ISSUE TITLE
Fetcher Access Modifiers

### ISSUE THREAD

**Issue by yoller** [Haneke/HanekeSwift#223]
In Fetcher type definition, init(), fetch() and cancelFetch() have 'internal' default access:

```Swift
public class Fetcher<T : DataConvertible> {

    public let key: String
    
    init(key: String) {
        self.key = key
    }
    
    func fetch(failure fail: ((NSError?) -> ()), success succeed: (T.Result) -> ()) {}
    
    func cancelFetch() {}
}
```

Why are they not public in a way I can create my own fetcher in my module just importing Haneke and inheriting Fetcher<T>?

If not publics I can't call super.init(key: key) at the end of my custom fetcher's init and I can't override func fetch() and func cancelFetch()

What's worng, maybe I have to implement my custom fetcher in another way?

**Comment by hpique** [comment]
My bad @yoller. Want to send a pull request?

**Comment by hpique** [comment]
Fixed in #224.
